draft-ietf-scitt-architecture.md

```diff
@@ -495,13 +490,14 @@ Might dereference to:
 
 ### Naming Artifacts
 
-Many Issuers issue Signed Statements about different Artifacts under the same DID, so it is important for everyone to be able to immediately recognize by looking at the Envelope of a Signed Statements what Artifact it is referring to.
-This information is stored in the Feed header of the Envelope.
+Many Issuers issue Signed Statements about different Artifacts under the same issuer identifier,
```

Comment:
The existing specification statements here are extremely confusing to me!

I think, we should clearly distinguish the two use cases and describe them individually.

Use Case 1:
An Issuer : X is issuing statements about  Multiple products (P, Q, R):

Use Case 2:
Multiple Issuers, X, Y,Z are making statements about the same product P.

We need to have these two cases clearly covered in the specificiation.

Reply:
> Multiple Issuers, X, Y,Z are making statements about the same product P.

This resonates with me, as it accomplishes the scenarios entities do today. 

> An Issuer : X is issuing statements about Multiple products (P, Q, R):

This one is less clear. Can we articulate how the above could be used in practice? Trying to correlate across multiple Artifacts (products as noted above) feels like a conflation and complication. Rather than attempt to make a statement about multiple artifacts, why not make the statement on each artifact?

If we can ground the designs in the problems we're trying to solve, ([the use cases](https://www.ietf.org/archive/id/draft-ietf-scitt-software-use-cases-01.html)) we might be able to better address a proposal. 
From: [Use CWT Claims in Headers: comment](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/108#issuecomment-1753959506)

The thing I'd like to test with this discussion is whether two or more identities can make statements about the same artifact.

1. Wabbit Networks (_the identity `391: did`_) releases their `net-monitor v1` software (_the Artifact_)
1. Wabbit Networks publishes a Signed Statement using a `"392: feed": "wabbitnetworks-netmonv3linux` and `"391: DID":="wabbitnetworks"` on their `scitt.wabbit-networks.io` service. 
1. Wabbit Networks has an auditor publish a signed statement: `"392: feed": "wabbitnetworks-netmonv3linux` and `"391: did":="coyote-security"` on the same `scitt.wabbit-networks.io` service.
1. Cosmic security, Mitre, and other independent publishers make statements about the same Artifact
    - `"392: feed": "wabbitnetworks-netmonv3linux` and `"391: did":="cosmicsecurity"` on their `scitt.cosmic-security.io` service. 
    - `"392: feed": "wabbitnetworks-netmonv3linux` and `"identity":="mitre"` on their `scitt.mitre.org` service. 

As a consumer, ACME Rockets can
- query the `scitt.wabbit-networks.io` service, to get statements associated with the net-monitor v1 software package.
- query `scitt.mitre.org` for any public statements, using `"392: feed": "wabbitnetworks-netmonv3linux` as the means to find statements from Mitre for the Wabbit Networks, net-monitor v1 software.
- optionally query `scitt.cosmic-security.io` as they specialize in aerospace software, providing deeper insights for the specific aerospace vertical.

ACME Rockets (_the consumer_) can choose to trust the statements because they trust the independent identities making statements. Which endpoints they query is up to them. They could equally choose to NOT trust Coyote Security, (because who trusts the Coyote?), or because they're not tailored to the unique needs Cosmic Security addresses.

In the above case, `392: feed` is used to independently identify a collection of statements about an artifact, independently from the identity, and consistent across SCITT Transparency Service Instances

Is the above usecase one we're trying to solve? 
- If not, what is the problem we're solving, and how would someone implement the above?
- If so, does `392: feed` and `391: did` solve the problem? If not, what are the alternatives?